app/api/tests/test_utils.py

```diff
@@ -143,13 +143,16 @@ def test_make_annotations(self):
 
 class TestCoNLLParser(TestCase):
     def test_calc_char_offset(self):
-        words = ['EU', 'rejects', 'German', 'call']
-        tags = ['B-ORG', 'O', 'B-MISC', 'O']
+        f = io.BytesIO()
 
-        entities = get_entities(tags)
-        actual = CoNLLParser.calc_char_offset(words, tags)
+        s = [
+            ("EU", "ORG"), ("rejects", "_"), ("German", "MISC"), ("call", "_")
+        ]
+        for w, t in s:
+            f.write("{}\t{}\n".format(w, t).encode())
+        f.seek(0)
```

Comment:
In order to avoid these one-letter variable names, I wonder if it would be simpler to just hard-code the test string like this?

```py
f = io.BytesIO(
  b"EU\tORG\n"
  b"rejects\t_\n"
  b"German\tMISC\n"
  b"call\t_\n"
)
```

Reply:
Yes, it is really prettier